schema.ts

```diff
@@ -242,6 +242,12 @@ interface BaseSchema {
   /** @hidden */
   mutable?: boolean;
 
+  /**
+   * Optional fixed id for this property, used to support renames of properties over time. If specified,
+   * changes to the name of this property will not cause the property to be treated as a new property.
+   */
+  fixedId?: string;
```

Comment:
This, `mutable`, and `autocomplete` only apply to top-level properties, not nested properties. I had David ensure that the documentation for `autocomplete` was clear on that, we should probably do the same for `fixedId` and `mutable`.

Reply:
Can you point me to that documentation?

Reply:
Based on conversation with David, it sounds like there isn't any documentation about this, at least not yet.

Reply:
I can add a quick note to this effect.